python/py-kaptan/Portfile

```diff
@@ -13,17 +13,15 @@ license             BSD
 maintainers         {@egorenar posteo.net:egorenar-dev} openmaintainer
 
 description         Configuration parser.
-long_description    ${description}
-
+long_description    Configuration parser with dict, JSON, YAML, .ini \
+                    and python file handlers.
 homepage            https://github.com/emre/kaptan
-master_sites        pypi:k/${python.rootname}/
-distname            ${python.rootname}-${version}
 
 checksums           rmd160  f815b558ccb87e79a9240f40ba65a23313a6e25b \
                     sha256  1abd1f56731422fce5af1acc28801677a51e56f5d3c3e8636db761ed143c3dd2 \
                     size    10539
 
-python.versions     27 35 36 37 38
+python.versions     27 35 36 37 38 39
 
 if {${name} ne ${subport}} {
     depends_lib-append \
```

Comment:
in this case it should remain `depends_lib` as it is also required at runtime, because the port [uses](https://github.com/emre/kaptan/blob/f4256ab1a06e05edfe0a5cb12ef173083eb30d7b/setup.py#L56) `entry_points`

Reply:
Ah ok thanks @reneeotten, I didn't realise that it had `entry_points`. I should've remembred that pitfall from a very useful comment you wrote https://github.com/macports/macports-ports/pull/6314#issuecomment-585490282.